R/mpf_exporters.R

```diff
@@ -510,7 +538,8 @@ export_as_pdf <- function(x,
                           min_siblings = 2,
                           font_family = "Courier",
                           font_size = 8,
-                          fontsize = lifecycle::deprecated(),
+                          fontsize = font_size,
+                          lineheight = 1.2, ## XXX this matches legacy behavior but differs from default everywhere else
```

Comment:
should we have a system getter for this default?

Reply:
If you give a fontspec it will override this argument, I guess we could have a "if there is a font device open grab its font_spec" getter. I'm leery of that much state though. i don't even like how much state I *had* to put in already